CONTRIBUTING.adoc

```diff
@@ -109,14 +109,46 @@ TIP: If you're unfamiliar with the AsciiDoc syntax, please consult this link:htt
 This repository holds the central documentation for the Jenkins project. The
 documentation can be broken down into three categories:
 
-. *Knowledge-base/Handbook* - rich and in-depth documentation, separated into
-  chapters covering a topic area, conceptually similar to the
-  link:https://www.freebsd.org/doc/en_US.ISO8859-1/books/handbook/[FreeBSD Handbook]
-. *Getting Started guides* - topic-specific, progressive documentation walking
-  the reader through the topic
+. *Jenkins User Documentation* - for people who want to _use_ Jenkins's existing
+  functionality and plugin features. This consists of the following parts:
+** *Tutorials* - step-by-step guides that teach users relatively new to
```

Comment:
Rather than adding these to this list, how about creating a another section `==== User Documentation` and have this sub-list in that section?

Reply:
Thanks @bitwiseman - that's a good idea - otherwise, the high-level **Adding documentation** overview will become a little too unwieldly (especially if/as we add more documentation categories to jenkins.io).

Incidentally, to help me complete this, perhaps you and/or @rtyler could elaborate on what the **Getting Started guides** (mentioned on this page) are actually in reference to (since this pre-dates my editing of any content on jenkins.io)? Looking at the Jenkins User Documentation as it stands now, I couldn't clearly see what type of documentation this is meant to cover. There's:
1. A [Getting Started with Jenkins](https://jenkins.io/doc/book/getting-started/) page (at the start of the User Handbook),
1. A [Getting Started with Pipeline](https://jenkins.io/doc/book/pipeline/getting-started/) page (immediately within the Pipeline chapter of the User Handbook),
1. A [Getting Started with Blue Ocean](https://jenkins.io/doc/book/blueocean/getting-started/), and
1. A [Getting Started with the Guided Tour](https://jenkins.io/doc/pipeline/tour/getting-started/) page (top page of the Guided Tour).

My hunch is that they're meant to be introductory sections to each new chapter of the User Handbook (like 2 and 3 above)?

Reply:
@gilesgas  if you drill into the git blame for this file, you'll see that this part of this doc is very old:

https://github.com/jenkins-infra/jenkins.io/blame/master/CONTRIBUTING.adoc#L112
leads to this commit from March 2016:
https://github.com/jenkins-infra/jenkins.io/commit/ffe9cde836abbf852f97325d87d937b09788298f

That means that the text predates _all_ of recent documentation efforts.  The pages you linked to fit the description but I'm not sure how that fits in our current model of documentation (https://www.divio.com/en/blog/documentation/).  Looks like they'd be tutorials or how-to guides.

Reply:
Thanks for the explanation @bitwiseman , sorry for leaving this PR for a bit, which I'm wrapping up now. Based on our current documentation model, it looks like some of the Getting Started pages I've linked to above are a bit of a mish-mash of "how-to guides" and "discussion" material to User Handbook chapters.

Continuing with the current documentation model, I'd like to keep the User Handbook as predominantly "reference" material, accompanied by relevant "discussion" (i.e. background/overview material) and "how-to guides" (i.e. small general procedures), which are specific to the given chapter/topic subject matter.

I'd like to re-purpose the "Getting Started ..." section descriptions on this page to refer to "how-to guides" (which will be the missing documentation model category here). Unlike the "how-to guides" in the User Handbook, the purpose of these "how-tos" is to describe procedures that resolve specific use case scenarios which are beyond the more general scope of the User Handbook but also do not hand-hold the reader through very specific scenarios (i.e. for the purposes of teaching - like what the tutorials do). We don't have any such "how-to guides" yet but as I re-work this documentation, I'm sure I'll find some good candidates.